src/Sunny.jl

```diff
@@ -101,9 +98,18 @@ include("SunnyGfx/SunnyGfx.jl")
 include("SunnyGfx/CrystalViewer.jl")
 export view_crystal, offline_viewers, browser
 
-include("ParallelTempering.jl")
+include("MonteCarlo/Samplers.jl")
+export propose_uniform, propose_flip, propose_delta, @mix_proposals, LocalSampler
+
+include("MonteCarlo/BinnedArray.jl")
+export BinnedArray, reset!, get_keys, get_vals, get_pairs
+
+include("MonteCarlo/ParallelTempering.jl")
 export ParallelTempering, sample!
 
+include("MonteCarlo/HistogramReweighting.jl")
+export WHAM, ensemble_average
```

Comment:
Consider removing these exports also

Reply:
I also intended for WHAM() and ensemble_average() to be called by users themselves after running PT, as they can also choose to not use histogram reweighting. The ensemble_average function can be used for arbitrary observables that are defined by the user and collected with BinnedArrays as an average for each energy level, which are then reweighted to any desired temperatures. 

The ensemble_average function will also be useful for users to calculate results with g(E) from wang-landau and multicanonical simulations once I put those codes back in.